server.js

```diff
@@ -0,0 +1,7 @@
+var express = require('express');
+var db = require('./config/db.js')
+
+var app = express();
+app.listen(8000);
+
```

Comment:
Add a console log so that when other parties download the app, they are told what port to go to for localhost in their browser.